PSI_Project/Models/Goal.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
```

Comment:
Why do we need two separate classes: Goal and SubjectGoal? Can't we store everything in one class?

Reply:
This is needed because Each subjectGoal will be in one Goal as a list, and each subjectGoal will have it's own fields, so the two classes are needed in this scenrario.